### PR TITLE
Fix update to 8.0.2

### LINKED ISSUE #86: Update to 8.0.2: Cannot redeclare class OC\Security\SecureRandom
Got stuck on step 3 "replace".
Admin log entry said:
"Cannot redeclare class OC\Security\SecureRandom at /MY_WWW_PATH/owncloud/lib/private/security/securerandom.php#24"

<hr/>
**Edit by @LukasReschke: A workaround can be found at https://github.com/owncloud/updater/issues/86#issuecomment-99187323**

**Edit by @VicDeo: And final fix is https://github.com/owncloud/updater/issues/86#issuecomment-85592285**

### PR DESCRIPTION
**Pull request by ghost**
This might fix https://github.com/owncloud/updater/issues/86#

### CODE REVIEW

**Comment by karlitschek** [comment]
@LukasReschke @DeepDiver1975 what do you think? It seems that this broke some 8.0.2 upgrades

File: lib/private/security/stringutils.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ public static function equals($expected, $input) {
 			return hash_equals($expected, $input);
 		}
 
-		$randomString = \OC::$server->getSecureRandom()->getLowStrengthGenerator()->generate(10);
+		$randomString = \SecureRandom::$server->getSecureRandom()->getLowStrengthGenerator()->generate(10);
```
Comment (DeepDiver1975) [inline lib/private/security/stringutils.php]: How could that solve anything? the class SecureRandom has no static member $server.

**Comment by DeepDiver1975** [comment]
:-1:

**Comment by fcturner** [comment]
Huh? I thought it was about calling a class, as this is the way it works in an app I am creating. And it worked for me with this edit...

**Comment by fcturner** [comment]
Isnt $server a variable which gets a value from getSecureRandom()?

**Comment by fcturner** [comment]
Just new and trying to understand :)

**Comment by LukasReschke** [comment]
`OC::$server` is a singleton that contains the service list. So when requesting `\OC::$server->getConfig()` the config instance is returned. What you did is pretty much doing nothing at all.

**Pull request closed without merging**